utils/k8s/k8s.go

```diff
@@ -259,15 +259,20 @@ func (k8s K8S) YAMLApply(yamlPath string) error {
 	return nil
 }
 
-// ExecToPodThroughAPI uninterractively exec to the pod with the command specified.
```

Comment:
I am not sure about this word 'uninterractively', should be something like 'non-interactive'

Reply:
You can write something like "ExecToPodThroughAPI performs non-interactive exec to the pod with the specified command".